lib/domain/interactor/dashboard/fetch_feed_interactor.dart

```diff
@@ -1,4 +1,14 @@
 import 'package:flutter_dasher/common/model/tweet.dart';
+import 'package:flutter_dasher/domain/interactor/dashboard/fetch_feed_interactor_impl.dart';
+import 'package:flutter_dasher/domain/repository/feed_repository.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'fetch_feed_interactor.g.dart';
+
+@riverpod
+FetchFeedInteractor fetchFeedInteractor(FetchFeedInteractorRef ref) {
+  return FetchFeedInteractorImpl(ref.read(feedRepositoryProvider));
+}
```

Comment:
I'm perfectly fine with this being here. I would also be fine with code like this replacing the code in inject dependencies :)